lib/elixir/lib/option_parser.ex

```diff
@@ -499,4 +562,28 @@ defmodule OptionParser do
   defp negative_number?(arg) do
     match?({_, ""}, Float.parse(arg))
   end
+
+  defp format_errors(errors, opts) do
+    details = Enum.map_join(errors, ". ", &format_error(&1, opts))
```

Comment:
What if we format this so that the output looks like

```
** (OptionParser.InvalidOptionError) 2 errors found: 

  * option --verbose is of the wrong type, expected a integer, given "true"
  * option --source is of the wrong type, expected a integer, given "lib"
```

? I think it looks clearer and easier to read and understand.